GameFiles/GameInterface/_integration_t6.gsc

```diff
@@ -23,6 +23,8 @@ Setup()
     
     RegisterClientCommands();
     
+    _SetDvarIfUninitialized("sv_iw4madmin_autobalance",0);
```

Comment:
minor complaint here but could you leave a space after the comma and also space out the parameters like in the rest of the file?